components/DropdownLight/DropdownLight.jsx

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import { colors, spacing, baseFontSize } from '../shared/theme';
+
+const itemPropType = PropTypes.oneOfType([
+  PropTypes.string,
+  PropTypes.shape({
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    label: PropTypes.string,
+  }),
+]);
+
+const DropdownSelect = styled.select`
+  width: 100%;
+  height: 44px;
+
+  ${({ theme }) => {
+    const { baseFontSize: baseFontSizeDropdownSelect } = theme;
+
+    return css`
+      font-size: ${baseFontSizeDropdownSelect}px;
+    `;
+  }}
+`;
+
+const DropdownLight = ({ disabled, items, theme }) => (
+  <>
+    <DropdownSelect disabled={disabled} theme={theme}>
```

Comment:
Suggestion: i missed the placeholder to inform "select a option", because in this way, the dropdown renders with the first item of object items as selected. 
In this case, all the dropdowns renders with "Lemon" selected.

this component
![image](https://user-images.githubusercontent.com/3389749/208741684-a8f77d45-fa0e-4f4a-b943-497bf07d000b.png)

Original 
![image](https://user-images.githubusercontent.com/3389749/208741785-35bb385c-ac2f-459b-a403-1f029b6e367e.png)

its only a suggestion, we could create a new task to implement it.